cvat-sdk/cvat_sdk/core/proxies/tasks.py

```diff
@@ -65,11 +65,13 @@ class Task(
 
     def upload_data(
         self,
-        resource_type: ResourceType,
```

Comment:
This is a breaking change, and it should be noted in the changelog.

Reply:
Agree., added.

Reply:
You didn't list the breaking part, which is that `resource_type` is now keyword-only.

Reply:
It's not the only broken aspect, because the arg is also moved. I don't see big use in such mention for users, because their interpreters should report this.

Reply:
> It's not the only broken aspect, because the arg is also moved.

Isn't that the same thing? It's moved into the keyword-only section.

> I don't see big use in such mention for users, because their interpreters should report this.

Ideally, users should discover breaking changes _before_ their code breaks, which they can do by reading the changelog.

Reply:
I'd expect that any line in the Changed and Removed sections of the change log could break something. I can add "This change breaks the API, you will need to adapt your code" if it makes the situation more clear.

Reply:
Currently it just says that `resource_type` has a default now - I wouldn't expect that to break my code. It should also say that the argument is now keyword-only.

Reply:
Ok, done.